build/images/cloudinit/id_rsa.capi

```diff
@@ -0,0 +1,27 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Perhaps we should call this key pair something like `id_rsa.insecure-capi` or something similar, to explicitly say that its private key is available on a public forum

Reply:
Eh, I'm not overly concerned about the name. The key isn't baked into the image. If anything, we can add a section to the README in a future update to describe the test `cloudInit` directory and its contents.

Reply:
fair enough

Reply:
See 79dbab555d8fc6ab39b2cf1be3339cce2f3ecabb for the doc updates mentioned above.